cmd/s2i/main.go

```diff
@@ -155,7 +155,12 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 			builder, _, err := strategies.GetStrategy(cfg)
 			s2ierr.CheckError(err)
 			result, err := builder.Build(cfg)
-			s2ierr.CheckError(err)
+			if err != nil {
+				fmt.Println("Build failed")
+				s2ierr.CheckError(err)
+			} else {
+				fmt.Println("Build success")
```

Comment:
should be using glog, not fmt.

success text should be "Build completed successfully"